lib/src/core/audit_logs/audit_log.dart

```diff
@@ -52,9 +79,21 @@ class AuditLog implements IAuditLog {
     raw["audit_log_entries"].forEach((o) {
       entries[Snowflake(o["id"] as String)] = AuditLogEntry(o as RawApiMap, client);
     });
+
+    raw['auto_moderation_rules'].forEach((o) {
+      autoModerationRules[Snowflake(o['id'] as String)] = AutoModerationRule(o as RawApiMap, client);
```

Comment:
as string not needed here since Snowflake accepts dynamic and casts to string byt itself